include/openspace/data/dataloader.h

```diff
@@ -22,20 +22,27 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
-#ifndef __OPENSPACE_MODULE_SPACE___SPECKLOADER___H__
-#define __OPENSPACE_MODULE_SPACE___SPECKLOADER___H__
+#ifndef __OPENSPACE_CORE___DATALOADER___H__
```

Comment:
Should this file maybe actually be called "Point Data Loader"? 

Since it's so specific to point-based datasets?